sehee/routes/index.js

```diff
@@ -0,0 +1,14 @@
+// 경로 작성
+const express = require("express");
+const router = express.Router();
+
+const userRouter = require("./userRouter");
+router.use("/user", userRouter.router);
+
+const postsRouter = require("./postsRouter");
+router.use("/posts", postsRouter.router);
+
+const likesRouter = require("./likesRouter");
+router.use("/likes", likesRouter.router);
```

Comment:
특정 미들웨어를 적용하는 코드 바로 윗줄에 그에 대한 변수를 할당하는 코드를 작성하는 것도 좋은 방법입니다.

다만, 아래와 같이 작성하면 미들웨어에 사용될 변수 모음 / 미들웨어를 적용하는 코드 모음으로 구분하여 구조를 한눈에 볼 수 있다는 장점이 있습니다.

```suggestion
const userRouter = require("./userRouter");
const postsRouter = require("./postsRouter");
const likesRouter = require("./likesRouter");

router.use("/user", userRouter.router);
router.use("/posts", postsRouter.router);
router.use("/likes", likesRouter.router);
```